docs/index.html

```diff
@@ -150,164 +150,165 @@ <h1 class=".page-header">Available Items</h1>
       <div class="grid grid-container" id="mainGrid">
 
         <div class="grid-sizer"></div>
-            <div class="grid-item">
-                <a href="#">
-                    <div class="thumbnail">
-
-                        <img src="images/couch.jpg" class="img img-thumbnail"/>
-                    </div>
-                </a>
-            </div>
+          
+        <div id="couch.jpg" class="grid-item">
```

Comment:
What's the plan here? You can't have the grid items all hardcoded into the page because it's going to change all the time, no?